x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -537,6 +540,125 @@ const IntegrationSettings = ({ onChange, fields }: IntegrationInfoFieldsProps) =
   </div>
 );
 
+const useEnsureDefaultNamespace = ({
```

Comment:
here to line 661 did not change, i just moved it to the top because this file is hard enough to read, having utility functions both at the top of the file and at its bottom was making it really confusing